knimin/lib/data_access.py

```diff
@@ -3040,15 +3066,20 @@ def create_sample_plate(self, name, plate_type_id, email=None,
         with TRN:
             self._sample_plate_is_unique(name)
             self._plate_type_exists(plate_type_id)
-            if email:
-                self._email_exists(email)
+            self._email_exists(email)
             sql = """INSERT INTO pm.sample_plate (name, plate_type_id, email,
-                                                  created_on, notes)
-                     VALUES (%s, %s, %s, %s, %s)
+                                                  created_on)
+                     VALUES (%s, %s, %s, now())
```

Comment:
I thought about it, but given that we are unsure on how to transfer the old data to the new DB structure I thought on simplifying the API as much as possible. When used through the interface this should be set automatically to the current time.